src/ftxui_sample/main.cpp

```diff
@@ -6,9 +6,9 @@
 #include <random>
```

Comment:
Feel free to just strip all the example files (I've been removing them one by one) instead of reformatting them.